spec/features/manage_cmr/subscriptions/index_subscriptions_spec.rb

```diff
@@ -130,6 +130,10 @@
 
       clear_cache
 
+      # CMR does a check to ensure the user 'fakeid' exists in EDL.
+      # So add this user to local cmr.
+      cmr_client.add_users_to_local_cmr(['fakeid'], nil)
```

Comment:
I think it makes more sense to do this in the cmr_client method that creates or updates a subscription. See https://github.com/nasa/mmt/blob/76f38989f3790692e85009f6711274d1a4711e7f/lib/cmr/cmr_client.rb#L578, for adding members to groups. Based on the error and the similarity of this issue to groups, I think that a developer working on subscriptions in development will also likely run into this issue and may not understand what to do. The comments in the code linked to might also be improved to that effect.

Reply:
That seems cleaner, will change, thanks.